generator/src/routes/+page.svelte

```diff
@@ -1,59 +1,243 @@
+      <div class="actions-wrapper">
+        <Button title="Generate IOXIO Tag" onClick={() => {}} />
+      </div>
+    </form>
+  </div>
+  <!-- Right Panel -->
+  <div class="qrcode-area-wrapper">
+    <div class="qrcode-area">
+      <div class="qrcode">
+        <img class="frame" src={subtractSvg} alt="" />
+        <img class="effect" src={effectSvg} alt="" />
+        <img class="logo" src={logoTagsSvg} alt="" />
+      </div>
+    </div>
+    <div class="status">IOXIO Tag generator</div>
+    <div class="product-area">
+      <div class="description">
+        <p>
+          This application serves as a demonstration platform for generating QR codes specifically
+          designed for Data Product Passports.
+        </p>
+        <p>
+          Utilizing the IOXIO Dataspace technology, this app showcases how QR codes can be generated
+          to serve as digital passports for various Data Products. These QR codes can be read using
+          a compatible reader application, enabling access to relevant product details in real time.
+        </p>
+        <a class="documentation" href="/">See documentation →</a>
+      </div>
+    </div>
+    <div class="footer">
+      <span>Made by</span>
+      <img src={logoSvg} alt="" />
+    </div>
+  </div>
+</div>
 
-  <Counter />
-</section>
-
-<style>
-  section {
+<style lang="scss">
+  .container {
     display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-    flex: 0.6;
-  }
-
-  h1 {
+    flex-direction: row;
+    margin: auto;
     width: 100%;
+    min-height: 100vh;
+    overflow: hidden;
+    .form-wrapper {
+      padding: 7.5rem;
+      background-color: white;
+      width: 55%;
+      position: relative;
+      @media screen and (max-width: 1440px) {
+        flex: 1;
+        padding: 3rem;
+      }
+      @media screen and (max-width: 1000px) {
+        width: 100%;
+      }
+      .title {
+        color: rgba(16, 25, 32, 1);
+        font-size: 2rem;
+        font-weight: 700;
+        margin-bottom: 4rem;
+      }
+      form {
+        position: relative;
+      }
+      .actions-wrapper {
+        margin-top: 5rem;
+      }
+      .logomarkSvg {
+        width: 30%;
+        position: absolute;
+        bottom: 2rem;
+        left: -2%;
+        z-index: 0;
+      }
+    }
+    .qrcode-area-wrapper {
+      padding: 1rem 3rem 1rem;
+      background-color: #111920;
+      width: 45%;
+      display: flex;
+      flex-direction: column;
+      @media screen and (max-width: 1440px) {
+        flex: 1;
+        padding: 1rem 3rem;
+      }
+      @media screen and (max-width: 1000px) {
+        width: 100%;
+      }
+      .qrcode-area {
+        flex: 1;
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        position: relative;
+        min-height: 26rem;
+        .frame {
+          width: 20rem;
+          height: 20rem;
+          position: absolute;
+          left: -2rem;
```

Comment:
More of a negative values

Reply:
![image](https://github.com/ioxiocom/ioxio-tags/assets/137529618/0a015335-893b-4954-b6a8-6af4e1a35686)

Its from design